buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -590,72 +582,52 @@ class BuildPlugin implements Plugin<Project>  {
      * @param project to be configured
      */
     private static void configureIntegrationTestTask(Project project) {
-        Jar hadoopTestingJar = project.rootProject.tasks.findByName('hadoopTestingJar') as Jar
-        if (hadoopTestingJar == null) {
-            // jar used for testing Hadoop remotely (es-hadoop + tests)
-            hadoopTestingJar = project.rootProject.tasks.create('hadoopTestingJar', Jar)
-            hadoopTestingJar.dependsOn(project.rootProject.tasks.getByName('jar'))
-            hadoopTestingJar.classifier = 'testing'
-            project.logger.info("Created Remote Testing Jar")
-        }
-
-        // Add this project's classes to the testing uber-jar
-        hadoopTestingJar.from(project.sourceSets.test.output)
-        hadoopTestingJar.from(project.sourceSets.main.output)
-        hadoopTestingJar.from(project.sourceSets.itest.output)
-
-        Test integrationTest = project.tasks.create('integrationTest', RestTestRunnerTask.class)
-        integrationTest.dependsOn(hadoopTestingJar)
-
-        integrationTest.testClassesDirs = project.sourceSets.itest.output.classesDirs
-        integrationTest.classpath = project.sourceSets.itest.runtimeClasspath
-        integrationTest.excludes = ["**/Abstract*.class"]
-
-        integrationTest.ignoreFailures = false
-
-        integrationTest.executable = "${project.ext.get('runtimeJavaHome')}/bin/java"
-        integrationTest.minHeapSize = "256m"
-        integrationTest.maxHeapSize = "2g"
-
-        integrationTest.testLogging {
-            displayGranularity 0
-            events "started", "failed" //, "standardOut", "standardError"
-            exceptionFormat "full"
-            showCauses true
-            showExceptions true
-            showStackTraces true
-            stackTraceFilters "groovy"
-            minGranularity 2
-            maxGranularity 2
-        }
+        if (project != project.rootProject) {
+            Jar itestJar = project.tasks.create('itestJar', Jar)
+            itestJar.dependsOn(project.tasks.getByName('jar'))
+            itestJar.getArchiveClassifier().set('testing')
+            project.logger.info("Created [${project.name}] Testing Jar")
```

Comment:
FYI, this is being logged at _configuration time_ as is. If we want to add this logging when we _actually_ build the jar we need to place it in a task action like `doLast`.

Reply:
I'll remove this log line, it's not really needed or helpful